### DISCUSSION TITLE
Welcome to Shepherd Discussions!

### DISCUSSION THREAD

**Discussion by RobbieTheWagner** [shipshapecode/shepherd#1268] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by artherrero** [comment]
Hey everyone! I'm curious if its possible to add html content within each step. I'd like to create an intro step that has some icons which would require adding some more html within the step. Any help would be greatly appriciated :) TY! @rwwagner90

**Reply by RobbieTheWagner** [reply]
Hi @artherrero, yes the text supports html. Is it not working for you?

**Reply by artherrero** [reply]
@rwwagner90 ok great! I actually have not tried yet wanted to ask before I spent anytime in the code again. Thanks for the info going to give it a try now 👍

**Comment by jatinS-dev** [comment]
Hey everyone! I have one issue in Shepherd JS,  I am working on one Guided Tour Project, How should I add Multi-page functionality to it?
@rwwagner90  Help needed!

**Reply by RobbieTheWagner** [reply]
@jatinS-dev this question has been asked and answered many times. Please read through the issues for some help. It depends on your app, but you have to have the tour live in some sort of global spot, hide the tour, then show it again on the next page or use `beforeShowPromise` to ensure your page transition is done before showing the next step.
